### PR TITLE
Implementation to work inside terminal.

### PR DESCRIPTION
**Pull request by jcs090218**
Working screen. What do you think?

working

### CODE REVIEW

File: company-quickhelp.el [hunk 1]
```diff
@@ -230,8 +240,7 @@ currently active `company' completion candidate."
   "Return t if and only if pos-tip is expected work in the current frame."
   (and
    (fboundp 'x-hide-tip)
-   (fboundp 'x-show-tip)
-   (not (memq window-system (list nil 'pc)))))
```
Comment (jcs090218) [inline company-quickhelp.el:234]: I don't know what this line does. But it prevent from running the popup tip.

File: company-quickhelp.el [hunk 2]
```diff
@@ -203,16 +204,25 @@ currently active `company' completion candidate."
...
+                  (pos-tip-show doc fg-bg (overlay-start ovl) nil timeout width nil
+                                (+ overlay-width overlay-position) 1)
+                (popup-tip doc :point (overlay-start ovl)
+                           :width width
+                           :nostrip t)))))))))
 
 (defun company-quickhelp--set-timer ()
   (when (or (null company-quickhelp--timer)
-        (eq this-command #'company-quickhelp-manual-begin))
+            (eq this-command #'company-quickhelp-manual-begin))
```
Comment (jcs090218) [inline company-quickhelp.el:225]: Some indentation issue? I guess.

**Review by jcs090218 — COMMENTED** [review]
Some note for what changes.

**Comment by jcs090218** [comment]
@expez Ping!

**Comment by expez** [comment]
Hi, thanks for opening a PR!

Unfortunately, as I've [mentioned](https://github.com/expez/company-quickhelp#what-isnt) in the readme, I don't want terminal support for this library.

Feel free to fork or just put this code in your `.emacs`.

**Pull request closed without merging**